syfertext/doc.py

```diff
@@ -154,15 +156,78 @@ def vector(self):
 
         return doc_vector
 
+    def get_vector(self, excluded_tokens: Dict[str, Set[object]] = None):
+        """Get document vector as an average of in-vocabulary token's vectors,
+        excluding token according to the excluded_tokens dictionary.
+
+        Args
+            excluded_tokens (Dict): A dictionary used to ignore tokens of the document based on values
+                of their attributes, the keys are the attributes names and they index, for efficiency, sets of values.
+                Example: {'attribute1_name' : {value1, value2},'attribute2_name': {v1, v2}, ....}
+
+        Returns:
+            doc_vector: document vector ignoring excluded tokens
+        """
+
+        # if the excluded_token dict in None all token are included
+        if excluded_tokens is None:
+            return self.vector
+
+        # enforcing that the excluded_tokens dict indexes, by the name of the attributes, sets of values.
+        excluded_tokens = {
```

Comment:
This comment is not very clear actually.

It should be:

Enforcing that the values of the  `excluded_tokens` dict are sets, not lists

Reply:
Ok, I agree, it's clearer written like this